lib/rules/max-nesting-depth/__tests__/index.js

```diff
@@ -110,6 +110,38 @@ testRule(rule, {
   ]
 });
 
+testRule(rule, {
+  ruleName,
+  config: [1, { ignore: ["blockless-pseudoclasses"] }],
+
+  accept: [
+    {
+      code: "a { b { top: 0; }}"
+    },
+    {
+      code: "a { b { &:hover { top: 0; }}}"
+    },
+    {
+      code: "a { b { &:nest { &:nest-lvl2 { top: 0; }}}}"
+    }
+  ],
+
+  reject: [
+    {
+      code: "a { b { c { top: 0; }}}",
+      message: messages.expected(1)
+    },
+    {
+      code: "a { &:hover { b { top: 0; }}}",
```

Comment:
I believe this should be accepted as there is only one level here.

Reply:
You are right!
```
a {
  &:hover { // pseudo-class (parent depth level 0 taken)
    b { // nesting level 1
      top: 0;
    }
  }
}
```